src/components/Feedback.js

```diff
@@ -85,13 +85,16 @@ class Feedback extends React.Component {
   }
 
   handleSubmitError = err => {
```

Comment:
`handleSubmitError = err => {` to `handleSubmitError = ({ response }) => {` since it's the only param used?